src/vue/backend/src/views/children/advancedContentTab.vue

```diff
@@ -38,6 +45,9 @@ export default {
         for (const key in data) {
             this.$store.dispatch(`${this.module}/setItem`, { key, value: data[key] })
         }
+    },
+    mounted () {
+        this.formDisabled = false
```

Comment:
This does not need to be set in mounted, as the data attribute sets the default. 
*Repeat for all instances